index-scheduler/src/index_mapper/mod.rs

```diff
@@ -90,6 +90,8 @@ pub struct IndexStats {
     pub number_of_documents: u64,
     /// Size of the index' DB, in bytes.
     pub database_size: u64,
+    /// Size of the index' DB, in bytes.
```

Comment:
We probably want to update either this comment, the above one, or both, so that we can differentiate between the two values.

```suggestion
    /// Size taken by the used pages of the index' DB, in bytes.
    ///
    /// As the DB backend does not return to the disk the pages that are not currently used by the DB,
    /// this value is typically smaller than `database_size`.
```